contracts/Rates.sol

```diff
@@ -1,10 +1,15 @@
 /*
- Generic symbol / WEI rates contract.
+ Generic token / ETH rates contract.
  only callable by trusted price oracles.
  Being regularly called by a price oracle
-    TODO: trustless/decentrilezed price Oracle
+ Note: symbol is always the "pegged symbol" not the token's own symbol!
+    TODO: trustless/decentralized price Oracle
```

Comment:
we should remove these TODOs if you agree

Reply:
agree